Games/core/Meeting.js

```diff
@@ -638,7 +638,8 @@ module.exports = class Meeting {
                     (!this.multi && this.votes[member.id] == null) ||
                     (this.multi && selections.length < this.multiMin && selections.indexOf("*") == -1)
                 ) {
-                    this.game.vegPlayer(member.player);
+                    const playerIsKicklynched = this.actionName === "Village Vote" && this.finalTarget === member.id;
+                    this.game.vegPlayer(member.player, playerIsKicklynched);

```

Comment:
```suggestion
                    const isKickEliminated = this.actionName === "Village Vote" && this.finalTarget === member.id; 
                    if (!isKickEliminated) { 
                        this.game.vegPlayer(member.player); 
                    }
```

great point, but it takes me ages to understand. multi-line suggestions are (more) readable